docker/docker-compose.prod.yml

```
@@ -0,0 +1,25 @@
+version: "3.9"
+
+services:
+  server:
+    container_name: "${PROJECT_NAME}"
+    env_file: .env
+    image: ghcr.io/lukim-gather-server/server:${PROJECT_VERSION}
```

```suggestion
    image: ghcr.io/nepware-dev/lukim-gather-server:${PROJECT_VERSION}
```
Update image url on all case valid url for this project